example/flannel-toleration.yaml

```diff
@@ -8,3 +8,17 @@ spec:
     spec:
       tolerations:
       - operator: Exists
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: kube-system
+  name: kube-flannel-cfg
+data:
+  net-conf.json: |
+    {
+      "Network": "192.168.0.0/16",
```

Comment:
I modified  [the Network setting](https://github.com/cybozu-go/cke/blob/tolerate-sonobuoy/example/kube-flannel.yaml#L128) in the upstream manifest directly. Is it better to make it back to the [original](https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml#L128)?

Reply:
Not necessarily.  When we will update flannel manifests, it will automatically back to the upstream.